graphql/requirements-dev.txt

```diff
@@ -13,4 +13,4 @@ flake8==4.0.1
 black==21.12b0
 isort==5.10.1
 coverage==5.3
-pytest==6.1.2
+pytest==6.2.5
```

Comment:
looks like other lambdas use 6.1.2
Should we update those as well to stay consistent?

Reply:
I'd be in favor of that. Got https://github.com/pytest-dev/pytest/discussions/9195 in this project, not sure why.